go-controller/pkg/ovn/namespace.go

```diff
@@ -320,13 +321,25 @@ func (oc *DefaultNetworkController) getAllHostNamespaceAddresses() []net.IP {
 	} else {
 		ips = make([]net.IP, 0, len(existingNodes))
 		for _, node := range existingNodes {
+			var hostNetworkIPs []net.IP
 			if config.HybridOverlay.Enabled && util.NoHostSubnet(node) {
-				// skip hybrid overlay nodes
-				continue
-			}
-			hostNetworkIPs, err := oc.getHostNamespaceAddressesForNode(node)
-			if err != nil {
-				klog.Errorf("Error parsing annotation for node %s: %v", node.Name, err)
+				// during SDN live migration, add the SDN node GW IP to the host network address set.
```

Comment:
this change is very openshift-specific, not sure if it belongs upstream...

Reply:
Can we have d/s only patch in openshift repo? If yes, I can propose this PR to d/s.

Reply:
I think this should be a downstream specific patch, and the IP should only be added during SDN migration and removed after. Otherwise we would always do this for hybrid overlay and it might cause inconsistent behavior that would be hard in the future to root cause.